atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/cleaner/ScrubberTest.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
```

Comment:
We should copy the copyright text from any other file - there's more than this (I've done this)